requirements/constraints.txt

```diff
@@ -37,5 +37,9 @@ factory-boy<4.0
 # pytest version 6.0 causing test failures
 pytest<7.0
 
-# Python 3.5 is dropped in version 5.0
+# pinning it to latest release.
 celery<5.0
+
+
+# latest version dropped support for python3.5 but our sandboxes still using that.
+cryptography==3.2.1
```

Comment:
Shouldn't that already be covered by https://github.com/edx/edx-enterprise/blob/master/requirements/edx-platform-constraints.txt#L29 ?